base/div.jl

```diff
@@ -364,7 +364,66 @@ function div(x::T, y::T, ::typeof(RoundUp)) where T<:Integer
     return d + (((x > 0) == (y > 0)) & (d * y != x))
 end
 
-# Real
-# NOTE: C89 fmod() and x87 FPREM implicitly provide truncating float division,
-# so it is used here as the basis of float div().
-div(x::T, y::T, r::RoundingMode) where {T<:AbstractFloat} = convert(T, round((x - rem(x, y, r)) / y))
+function mw_div(x::NTuple{2,T}, y::T) where {T<:AbstractFloat}
```

Comment:
This probably belongs in Base.twiceprecision.jl

Reply:
NB: This was pointed out already above, in response to which I opened a PR to update twiceprecision.jl, #49569, but that PR's waiting on me to:

1. break it up, as it seems to have too wide a focus, probably I should separate the complex stuff from the real stuff

2. find examples where the new algorithms are more accurate than the current, presumably outdated algorithms

Reply:
fair enough